src/ott/problems/linear/potentials.py

```diff
@@ -258,3 +264,125 @@ def epsilon(self) -> float:
 
   def tree_flatten(self) -> Tuple[Sequence[Any], Dict[str, Any]]:
     return [self._f, self._g, self._prob, self._f_xx, self._g_yy], {}
+
+
+def plot_ot_map(
```

Comment:
Think this should be a method of the `DualPotentials`, since it takes it as a first argument (same for the other function).

Reply:
That makes sense, moved there